graph/builder.go

```diff
@@ -48,16 +47,21 @@ type IndexingBuilder interface {
 	OpenIndex() (librarian.SeqSetterIndex, librarian.SinkIndex)
 }
 
-type builder struct {
+type BadgerBuilder struct {
```

Comment:
rename to BendyBuilder? iirc there's not bipfy badger anymore

Reply:
nope, will clear this up with a comment.

This is about the [badger key-value db](https://github.com/dgraph-io/badger) that powers the index system.

Reply:
oh lol